README.md

```diff
@@ -36,7 +36,7 @@ If you encounter any issues while using Strimzi, you can get help using:
 ## Strimzi Community Meetings
 
 You can join our regular community meetings:
-* Thursday 8:00 AM UTC (every 4 weeks starting from 4th June 2020) - [convert to your timezone](https://www.thetimezoneconverter.com/?t=8%3A00&tz=UTC)
+* Thursday 9:00 AM UTC (every 4 weeks starting from 11th December 2025) - [convert to your timezone](https://www.thetimezoneconverter.com/?t=8%3A00&tz=UTC)
```

Comment:
I think the dates here are confusing. Should we just remove them? Both here and the line below? Given the time anyway applies already from the next call, they seem to be meaningless.